00-base/01-colors/colors.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import grayscale from './colors-grayscale.js';
+import branding from './colors-branding.js';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Base/Colors' };
+
+const Colors = ({ palette }) => (
+  <ul className="sg-colors">
+    {palette.map((color) => (
+      <li className="sg-colors__list" key={color}>
+        <div className={`sg-colors__color sg-colors__color--${color}`}></div>
```

Comment:
IDK if this is a good place to do it, but having some sort of classname composition utils could help you clean these things up a bunch, especially if you need repetitive strings like this often. That said, this is really simple, so it's nbd!